web/app.go

```diff
@@ -164,8 +164,10 @@ func NewAppWithDeps(config *Config, deps Dependencies) (*App, error) {
 	webEngine.Use(ErrorHandler)
 	webEngine.Use(sessions.Sessions("session", deps.store))
 	webEngine.StaticFS("/static", http.FS(assetsFS))
+	webEngine.Use(EulaMiddleware(deps.settingsService))
 	webEngine.GET("/", HomeHandler)
 	webEngine.GET("/about", NewAboutHandler(deps.subscriptionsService))
+	webEngine.GET("/accept-eula", EulaAcceptHandler(deps.settingsService))
```

Comment:
Wouldn't be more appropriate a `POST` or `PUT` methods as we will record a value in the database?

Reply:
If the button was inside a form, yes, but the button to accept is a simple link, so the only way to do this is using a `GET`. I didn't think about the form, but it would be a lot of stuff in the eula page markup to achieve basically the same result.

Reply:
I kinda agree that to be totally  RFC compliant, a GET should always be idempotent, while the POST usually has side-effects. One of the bad things that can generally happen is that some browsers trying to be smart and use prefetch of the links to cache nearest pages, this would mean that the EULA would be accepted without the user clicking on it.

Reply:
> If the button was inside a form
>
@dottorblaster you're right and indeed we could wrap the button in a form that will only have that button, and have it submitted by POSTing.
Do you see that possible? :thinking:

Reply:
Yeah @nelsonkopliku, I didn't want to end up with a lot of markup for that but in the end...

@fabriziosestito I don't think that prefetching is something that affects us, but I totally see the point. I'll address this ASAP, thanks and thanks @arbulu89 for noticing it